spec/controllers/api/v1/employees_controller_spec.rb

```diff
@@ -0,0 +1,207 @@
+# frozen_string_literal: true
```

Comment:
I'd split this spec to two: one where user is manager and another one where user is employee.